.github/workflows/grab_spotify_saved_tracks.yaml

```
@@ -0,0 +1,42 @@
+    update-spotify-tracks:
+        runs-on: ubuntu-latest
+        env:
+            SPOTIFY_CLIENT_ID: ${{ secrets.SPOTIFY_CLIENT_ID }}
+            SPOTIFY_CLIENT_SECRET: ${{ secrets.SPOTIFY_CLIENT_SECRET }}
+            SPOTIFY_REDIRECT_URI: ${{ secrets.SPOTIFY_REDIRECT_URI }}
+            SPOTIFY_TOKEN_CACHE: ${{ secrets.SPOTIFY_TOKEN_CACHE }}
+            SPOTIFY_TOKEN_CACHE_PATH: .spotify_cache
+
+        steps:
+            - name: Check out repository
+              uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)